src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1142,43 +1142,23 @@
     <Field Name="InitializerExpressionOpt" Type="BoundExpression" Null="allow"/>
   </Node>
   
-
-  <!-- 
-  Tuple literals can exist in two forms - literal and converted literal.
-  This is the base node for both forms.
-  -->
-  <AbstractNode Name="BoundTupleExpression" Base="BoundExpression">
-    <Field Name="Arguments" Type="ImmutableArray&lt;BoundExpression&gt;"/>
-  </AbstractNode>
-
   <!-- 
   Tuple literals can convert to the target type.
   Once converted to a target type, they cannot be target-typed again. 
   The tuple literal is one which has not been converted to a target type.
   -->
-  <Node Name="BoundTupleLiteral" Base="BoundTupleExpression">
+  <Node Name="BoundTupleLiteral" Base="BoundExpression">
     <!-- 
     It is possible for a tuple to not have a type in a literal form
     Ex: 
      (a: 1, b: (c: 1, d: null))  does not have a natural type, because "null" does not have one
     -->
-    <Field Name="Type" Type="TypeSymbol" Override="true" Null="allow"/>
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="always"/>
+    <Field Name="Arguments" Type="ImmutableArray&lt;BoundExpression&gt;"/>
     <Field Name="ArgumentNamesOpt" Type="ImmutableArray&lt;string&gt;" Null="allow"/>
+    <Field Name="DeclaredOrTargetTupleTypeOpt" Type="TupleTypeSymbol" Null="allow"/>
```

Comment:
This field needs a comment. 
It is not obvious what "DeclaredOrTargetTupleTypeOpt" represents